test/nightly/createProjectAndDeploy.test.ts

```diff
@@ -23,15 +23,18 @@ interface CreateProjectAndDeployTestCase extends ICreateProjectAndDeployOptions
 }
 
 const testCases: CreateProjectAndDeployTestCase[] = [
-    { title: 'JavaScript', ...getJavaScriptValidateOptions(true), createProjectInputs: [/Model V3/], deployInputs: [getRotatingNodeVersion()] },
-    { title: 'TypeScript', ...getTypeScriptValidateOptions(), createProjectInputs: [/Model V3/], deployInputs: [getRotatingNodeVersion()] },
+    { title: 'JavaScript (Model V3)', ...getJavaScriptValidateOptions(true), createProjectInputs: [/Model V3/], deployInputs: [getRotatingNodeVersion()], languageModelVersion: NodeModelVersion.v3 },
```

Comment:
Similar to what Alex mentioned before, we should have a constant for Model V3 and Model V4 inputs.

Reply:
Done :)